bin/config.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2020 Dmitry Ivanko. All Rights Reserved.
+# Copyright (C) 2020 Vladislav Yarovoy. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Configuration file of server.py"""
+import os
+
+PORT = os.getenv('PORT', '8005')
+MEDIAWIKI_URL = os.getenv('MEDIAWIKI_URL', '127.0.0.1:8004')
```

Comment:
Let's rename it into `MEDIAWIKI_API_URL`. In case of API URL the environment variable must include `api.php`.